table.js

```diff
@@ -111,16 +122,18 @@ class BigQueryTable {
         let items = Array.isArray(rows) === false ? [rows] : rows;
 
         return items.map(item => {
-            return { insertId: BigQueryTable.getInsertId(), json: item };
+            return BigQueryTable.getRawRow(item);
         });
     }
 
-    /**
-     * @return {String}
-     */
-    /* istanbul ignore next */
-    static getInsertId() {
-        return uuid.v4();
+    static getRawRow(row) {
+        let item = Object.assign({}, row);
```

Comment:
Cloning the item here is not needed. `insertData` is a new objet that the user that uses this library cannot handle